test/unit_tests/test_braket_device.py

```diff
@@ -1522,3 +1524,93 @@ def get_test_result_object(result_types=[], source="qubit[2] q; cnot q[0], q[1];
         }
     )
     return GateModelQuantumTaskResult.from_string(json_str)
+
+
+@pytest.fixture
+def noise_model():
+    return NoiseModel().add_noise(BitFlip(0.05), GateCriteria(Gate.H))
```

Comment:
I wonder if it would be more robust to test 1Q gate noise, 2Q gate noise and readout noise all together